fcl/dunefdvd/caf/standard_cafmaker_dunevd10kt_1x8x6_3view_30deg.fcl

```diff
@@ -0,0 +1,8 @@
+#include "standard_cafmaker_dune10kt_1x2x6.fcl"
```

Comment:
We've been avoiding having the VD fcls inherit from the HD fcls.  Could you copy the contents of the HD fcl here (but swap out the service block to VD 1x8x6 3view 3deg)?